### DISCUSSION TITLE
Fill in the grid

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1040] [Galaxy Trivia] [answered]
<img width="235" alt="image" src="https://github.com/ParanoidUser/yolo/assets/5120290/adb79bda-d919-4be8-b963-af8ba24aa6f6">

# Objective / Rules
 - Fill in the grid using numbers from 1 to 9.
 - Each number should be used only once.
 - The three vertical and three horizontal equations must be true.

**Comment by tomerf** [comment, marked as answer]
| 8  | - | 6  | + | 4 | = | 6  |
|----|---|----|---|---|---|----|
| +  |   | -  |   | - |   |    |
| 1  | + | 3  | - | 2 | = | 2  |
| +  |   | -  |   | + |   |    |
| 9  | - | 5  | - | 7 | = | -3 |
| =  |   | =  |   | = |   |    |
| 18 |   | -2 |   | 9 |   |    |
